open-ce/build_image.py

```diff
@@ -81,7 +81,10 @@ def build_runtime_docker_image(args):
     _validate_input_paths(local_conda_channel, conda_env_file)
 
     # Copy the conda environment file into the local conda channel to modify it
-    shutil.copy(conda_env_file, local_conda_channel)
+    try:
+        shutil.copy(conda_env_file, local_conda_channel)
```

Comment:
This copying was earlier added because conda env file location was different than the output_folder. 
We do want to copy but not to local_conda_channel as it will already have it. And copy is also must because we modify channels in conda env file according to runtime docker container, which would make the env file erroneous if someone uses it to create a conda environment on the host using it.

Reply:
yeah, I considered making a back up in this case, but there are some things to consider: 
 - We could move the original to SOMEWHERE/FILE.orig, but then if the user runs again, it will just be overwritten by the back up of the edited file.
 - So we could move the original to SOMEWHERE/FILE.orig.date (or something) but then we are accruing a bunch of garbage along side the one original as a new backup is made on each run. That's annoying an unhelpful.
 
It's also a bit unfortunate because it's very likely the env file that is being used will be located in the conda channel already because we put it there after the build. The behavior as it is now fails out on this SAMEFILE error and ends, so it must be handled. I decided to just not copy and this case which does lose the original, but..I think it's not so bad. The user can back it up (or manually edit it) if they need to go back and use the env file manually.

One other comment - we don't have or use a message logger in OpenCE yet (perhaps we should), so I'm just manually doing a print here.

Reply:
I think we can just create the copy of the env file in the same output_folder (local_conda_channel) with name like `opence-conda-env-py3.6-cpu-openmpi-10.2-docker-runtime.yaml` and edit that and pass it onto the docker image. This way, the original file will remain intact for user to work on host.

Reply:
yeah - that's doable.

Reply:
Yeah, after we are done with the image creation, we can delete this new copy. That way host won't have files that are useless for it.